Detectors/MUON/Common/src/dcs-processor-workflow.cxx

```diff
@@ -266,10 +266,16 @@ void processDataPoints(o2::framework::ProcessingContext& pc,
   }
   auto dps = pc.inputs().get<gsl::span<o2::dcs::DataPointCompositeObject>>("input");
   for (auto dp : dps) {
-    // FIXME: check we're not adding twice the same dp (i.e. check timestamp ?)
     for (auto i = 0; i < NOBJECTS; i++) {
       if (std::find(aliases[i].begin(), aliases[i].end(), dp.id.get_alias()) != aliases[i].end()) {
-        dataPoints[i][dp.id].emplace_back(dp.data);
+        auto& v = dataPoints[i][dp.id];
+        bool shouldAdd{true};
+        if (v.size() > 0 && v.back() == dp.data) {
+          shouldAdd = false;
+        }
+        if (shouldAdd) {
+          dataPoints[i][dp.id].emplace_back(dp.data);
```

Comment:
you could have used v.emplace_back(dp.data);

Reply:
indeed. now fixed.